utils/s2n_init.c

```diff
@@ -87,6 +87,10 @@ int s2n_init(void)
         s2n_stack_traces_enabled_set(true);
     }
 
+#if defined(OPENSSL_IS_BORINGSSL) || defined(OPENSSL_IS_AWSLC)
+    CRYPTO_pre_sandbox_init();
+#endif
```

Comment:
I debated putting this in a feature probe. I know it would be better, but it kind of seems like overkill :/ Thoughts?

Reply:
I guess it depends on a few things:

* Are there versions of these libraries that don't include this API and do we want to support those?
* Do we anticipate other libcryptos adding this API in the future?

Reply:
I wouldn't expect other libraries to add it. It's not from openssl.

The API has been in aws-lc from the beginning: https://github.com/aws/aws-lc/commit/3e4dfbb2f81493e1f25b3ce83fee3ed999373a6c (note that commit was merged in the first PR, along with the rest of the boringssl history).

But it's only been in boringssl since 2020: https://github.com/google/boringssl/commit/3e4dfbb2f81493e1f25b3ce83fee3ed999373a6c However, we don't really have an alternative, so earlier versions would just not be sandboxable. Still, not being sandboxable is better than not building at all. Let me see if I can find a reasonable place for documentation.

Reply:
Since we're not testing this on boringssl (and I doubt plan to), I restricted the call to aws-lc and documented the support.